integration_test/third_party_apps_data/applications/active_directory_ds/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+short_name: AD DS
+long_name: Active Directory Domain Services
+description: |-
+  Active Directory Domain Services stores information about objects on the network so that administrators and users
+  can easily access this information. For more information, see 
+  [Microsoft's overview](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/active-directory-domain-services-overview)
+configure_integration: |-
+  By default, Active Directory Windows event logs and performance counters are enabled.
+restart_after_install: true
+configuration_options:
+  logs:
+    - type: active_directory_ds
+      fields:
+      - name: type
+        default: null
+        description: The value must be active_directory_ds.
+expected_logs:
+- log_name: active_directory_ds
+  fields:
+  - name: jsonPayload.Message
+    value_regex: '^Microsoft Active Directory Domain Services startup complete.*'
+    type: string
+    description: "The log message."
+  - name: jsonPayload.RecordNumber
+    type: number
+    description: "The sequence number of the event log."
+  - name: jsonPayload.TimeGenerated
+    value_regex: '^\d+-\d+-\d+ \d+:\d+:\d+ [+-]\d{4}$'
+    type: string
+    description: "A timestamp representing when the record was generated."
+  - name: jsonPayload.TimeWritten
+    value_regex: '^\d+-\d+-\d+ \d+:\d+:\d+ [+-]\d{4}$'
+    type: string
+    description: "A timestamp representing when the record was written to the event log."
+  - name: jsonPayload.EventID
+    type: number
+    description: "An ID identifying the type of the event."
+  - name: jsonPayload.EventType
+    type: string
+    description: "The type of event."
+  - name: jsonPayload.Qualifiers
+    type: number
+    description: "A qualifier number that is used for event identification."
+  - name: jsonPayload.EventCategory
+    type: number
+    description: "The category of the event."
+  - name: jsonPayload.Channel
+    value_regex: '^Directory Service$'
```

Comment:
Does this need to be `value_regex: '^Directory Service|Active Directory Web Services$'`?

Reply:
My understanding is that all these regexes should be as specific as possible to match only a single entry.

I can add another test case for a log from the Active Directory Web Services channel, if that's the concern.

Reply:
I added in another case to ensure we get messages from the web services channel